pkgs/development/tools/build-managers/conan/default.nix

```diff
@@ -1,80 +1,73 @@
-{ lib, python3, fetchpatch, git }:
+{ lib, python3, git }:
 
 let newPython = python3.override {
   packageOverrides = self: super: {
     distro = super.distro.overridePythonAttrs (oldAttrs: rec {
-      version = "1.1.0";
+      version = "1.2.0";
       src = oldAttrs.src.override {
         inherit version;
         sha256 = "1vn1db2akw98ybnpns92qi11v94hydwp130s8753k6ikby95883j";
```

Comment:
You can't just change the version without changing the hash.

Reply:
Sorry but that version of the file was not the one merged.

https://github.com/NixOS/nixpkgs/blob/fd5b4eeb19154efd00195fac9f79583a4562ba1e/pkgs/development/tools/build-managers/conan/default.nix

Reply:
fixed in 2eaadfa5be7ce653a2bca31671c406a2ee7f7b0f

conan does not like 1.2.0 anyway.